src/Microsoft.Data.SqlClient/tests/ManualTests/DataCommon/DataTestUtility.cs

```diff
@@ -44,10 +44,11 @@ public static class DataTestUtility
         public static bool EnclaveEnabled { get; private set; } = false;
         public static readonly bool TracingEnabled = false;
         public static readonly bool SupportsIntegratedSecurity = false;
-        public static readonly bool SupportsFileStream = false;
         public static readonly bool UseManagedSNIOnWindows = false;
         public static readonly bool IsAzureSynapse = false;
         public static Uri AKVBaseUri = null;
+        public static string FileStreamDirectory = null;
```

Comment:
NIT: I suggest using property instead of a field here to verify its value.

Reply:
This value is tied to config.json, I didn't want to modify it a lot and keep design consistent with current properties.